generators/client/files-angular.js

```diff
@@ -305,6 +305,7 @@ const files = {
                 'admin/user-management/_user-management-detail.component.ts',
                 'admin/user-management/_user-management-dialog.component.ts',
                 'admin/user-management/_user-management-delete-dialog.component.ts',
+                'admin/user-management/_user-modal.service.ts',
                 'admin/user-management/_user.model.ts',
                 'admin/user-management/_user.service.ts',
                 { file: 'admin/user-management/_user-management.state.ts', method: 'copyJs' },
```

Comment:
Can you explain why the change ? Right now, without i18n, the titles are like `home.title` because we don't change the `pageTitle` field in the state files.